blas/unit_test/Test_Blas1_iamax.hpp

```diff
@@ -52,31 +52,31 @@ void impl_test_iamax(int N) {
     expected_max_loc = 0;
   }
 
-  {
-    // printf("impl_test_iamax -- return result as a scalar on host -- N %d\n",
-    // N);
-    size_type nonconst_max_loc = KokkosBlas::iamax(a.d_view);
-    ASSERT_EQ(nonconst_max_loc, expected_max_loc);
-
-    size_type const_max_loc = KokkosBlas::iamax(a.d_view_const);
-    ASSERT_EQ(const_max_loc, expected_max_loc);
-  }
-
-  {
-    // printf("impl_test_iamax -- return result as a 0-D View on host -- N
-    // %d\n", N);
-    typedef Kokkos::View<size_type, typename ViewTypeA::array_layout, Kokkos::HostSpace> ViewType0D;
-    ViewType0D r("Iamax::Result 0-D View on host", typename ViewTypeA::array_layout());
-
-    KokkosBlas::iamax(r, a.d_view);
-    Kokkos::fence();
-    size_type nonconst_max_loc = r();
-    ASSERT_EQ(nonconst_max_loc, expected_max_loc);
-
-    KokkosBlas::iamax(r, a.d_view_const);
-    size_type const_max_loc = r();
-    ASSERT_EQ(const_max_loc, expected_max_loc);
-  }
+  // {
+  //   // printf("impl_test_iamax -- return result as a scalar on host -- N %d\n",
+  //   // N);
+  //   size_type nonconst_max_loc = KokkosBlas::iamax(a.d_view);
+  //   ASSERT_EQ(nonconst_max_loc, expected_max_loc);
+
+  //   size_type const_max_loc = KokkosBlas::iamax(a.d_view_const);
+  //   ASSERT_EQ(const_max_loc, expected_max_loc);
+  // }
+
+  // {
+  //   // printf("impl_test_iamax -- return result as a 0-D View on host -- N
+  //   // %d\n", N);
+  //   typedef Kokkos::View<size_type, typename ViewTypeA::array_layout, Kokkos::HostSpace> ViewType0D;
+  //   ViewType0D r("Iamax::Result 0-D View on host", typename ViewTypeA::array_layout());
+
+  //   KokkosBlas::iamax(r, a.d_view);
+  //   Kokkos::fence();
+  //   size_type nonconst_max_loc = r();
+  //   ASSERT_EQ(nonconst_max_loc, expected_max_loc);
+
+  //   KokkosBlas::iamax(r, a.d_view_const);
+  //   size_type const_max_loc = r();
+  //   ASSERT_EQ(const_max_loc, expected_max_loc);
+  // }
```

Comment:
What happend here?

Reply:
Nothing, forgot to re-enable after changing the init method. I'll go ahead and reenable this and work on the next issue. There are still both compile time and runtime issues with 2025.